src/main/webapp/view/discussion/discussion-manager.js

```diff
@@ -114,26 +114,29 @@ export default class DiscussionManager {
   }
 
   /**
-   * Posts `content` to the discussion with the given `params`.  This method is
-   * private and should only be called within `DiscussionManager`.
+   * Posts `content` to the discussion with the given `paramNamesTo`.
+   * This method is private and should only be called within
+   * `DiscussionManager`.
    *
    * <p>Different types of comments require different parameters, such as
    * `PARAM_TIMESTAMP` or `PARAM_PARENT`. The caller should ensure the correct
    * parameters are supplied for the type of comment being posted.
    */
-  async postComment(content, params) {
+  async postComment(content, paramNamesTo) {
```

Comment:
param names to what?
In general you can name maps ```<keyName>To<Value>``` I believe you're missing the value.

Reply:
Gotcha, I updated it to be `commentParameterNameToValue`. I couldn't think of a better name for `value` because it's either an id or a string depending on the parameter.